doc/classes/TextServer.xml

```diff
@@ -53,6 +59,122 @@
+			<param index="2" name="transform" type="Transform2D" />
+			<param index="3" name="callback" type="Callable" />
+			<description>
+				Adds custom draw call to a draw list [param list]. [param callback] is called with one argument, [RID] of canvas item to draw it to.
+			</description>
+		</method>
+		<method name="draw_list_add_hexbox" qualifiers="const">
+			<return type="void" />
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="layer" type="int" />
+			<param index="2" name="transform" type="Transform2D" />
+			<param index="3" name="pos" type="Vector2" />
+			<param index="4" name="index" type="int" />
+			<param index="5" name="size" type="int" />
+			<param index="6" name="modulate" type="Color" />
+			<description>
+				Adds draw call for box displaying character hexadecimal code to a draw list [param list].
+			</description>
+		</method>
+		<method name="draw_list_add_line" qualifiers="const">
+			<return type="void" />
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="layer" type="int" />
+			<param index="2" name="transform" type="Transform2D" />
+			<param index="3" name="start" type="Vector2" />
+			<param index="4" name="end" type="Vector2" />
+			<param index="5" name="width" type="float" />
+			<param index="6" name="dash" type="float" />
+			<param index="7" name="modulate" type="Color" />
+			<description>
+				Adds draw call for a line to a draw list [param list].
+			</description>
+		</method>
+		<method name="draw_list_add_rect" qualifiers="const">
+			<return type="void" />
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="layer" type="int" />
+			<param index="2" name="transform" type="Transform2D" />
+			<param index="3" name="rect" type="Rect2" />
+			<param index="4" name="filled" type="bool" />
+			<param index="5" name="modulate" type="Color" />
+			<description>
+				Adds draw call for a rectangle to a draw list [param list].
+			</description>
+		</method>
+		<method name="draw_list_add_texture" qualifiers="const">
+			<return type="void" />
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="layer" type="int" />
+			<param index="2" name="transform" type="Transform2D" />
+			<param index="3" name="texture" type="RID" />
+			<param index="4" name="rect" type="Rect2" />
+			<param index="5" name="modulate" type="Color" />
+			<description>
+				Adds draw call for a texture to a draw list [param list].
+			</description>
+		</method>
+		<method name="draw_list_draw">
+			<return type="void" />
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="ci" type="RID" />
+			<param index="2" name="free" type="bool" default="false" />
+			<description>
+				Draws all draw calls in the draw list to the canvas item [param ci]. If [param free] is [code]true[/code], draw list is automatically deleted.
+			</description>
+		</method>
+		<method name="draw_list_reserve">
```

Comment:
Are we sure we want to expose what is essentially a `reserve` method to the public API? I would be very wary since this is a [very prickly topic of discussion](https://github.com/godotengine/godot/pull/110709#issuecomment-3314529135).
(Note that clayjohn got originally confused by the purpose of the linked PR).

Reply:
With the way `TextServer` is implemented (to be implementable as GDExtension), I do not think it is possible to have non-exposed methods. And unlike `Dictionary`, `TextServer` is a low level API most users won't ever interact with, so it should not be an issue.